Cloud Application Manager/Administering Your Organization/manage-assets-monitor-usage.md

```diff
@@ -1,29 +1,57 @@
+"tags", "webhooks", "authentication", "theme", "setting"],
 "author": "",
 "attachments": [],
 "contentIsHTML": false
 }}}
 
 ### Manage Cloud Application Manager Assets
+**In this article:**
 
-From the [admin console](admin-overview.md), you get full purview over assets others created in your organization. This includes details, administrators, clouds, tags, webhooks, authentication, and theme.
+* [Overview](#overview)
+* [Audience](#audience)
+* [Prerequisites](#prerequisites)
+* [Manage Details](#manage-details)
+* [Manage Administrators](#manage-administrators)
+* [Manage Clouds](#manage-clouds)
+* [Manage Tags](#manage-tags)
+* [Manage Webhooks](#manage-webhooks)
+* [Manage Authentication](#manage-authentication)
+* [Manage Theme](#manage-theme)
 
-**In this article:**
+### Overview
+
+This article makes a brief preview of the assets management section that Cloud Application Manager provides for any user with administrator privileges.
+
+### Audience
+
+All users with Cloud Application Manager organization administrator access.
 
-* Manage Details
-* Manage Administrators
-* Manage Clouds
-* Manage Tags
-* Manage Webhooks
-* Manage Authentication
-* Manage Theme
+### Prerequisites
+
+* Access to Cloud Application Manager, Management site.
+
+* The user must be an Administrator of the organization in Cloud Application Manager.
+
+* The user should be at the organization level scope to access the Settings option in the left side menu.
 
 
 ### Manage Details
 
-In the admin console, **Details** shows a list of items which describe the organization, as the icon, name, domain, account type, account status, etc. This items are not enabled.
+In the admin console settings, **Details** shows a list of items which describe the organization, as the icon, name, domain, account type, account status, etc. This items are not editable.
+To see more info about the options in this section you can refer to these pages:
+
+* [AWS Sourcing](https://www.ctl.io/knowledge-base/cloud-application-manager/cloud-optimization/partner-cloud-integration)
+* [Microsoft Azure Sourcing](https://www.ctl.io/knowledge-base/cloud-application-manager/cloud-optimization/partner-cloud-integration/)
+* [Managed Services Anywhere](https://www.ctl.io/cloud-application-manager/managed-services-anywhere)
+* [Application Lifecycle Management](https://www.ctl.io/cloud-application-manager/application-lifecycle-management)
+* [Monitoring](https://www.ctl.io/knowledge-base/cloud-application-manager/monitoring/cammonitoringui)
+* [Analytics](https://www.ctl.io/knowledge-base/cloud-application-manager/analytics)
+* [Network Exchange](https://www.ctl.io/knowledge-base/network/network-exchange-getting-started-guide)
+* Show Unregistered Resources: By enabling this option in the Settings, you'll be able to see the unregistered instances under the CAM/ALM Instances menu.
 
 ![admin-assets1.png](../../images/cloud-application-manager/admin-assets1.png)
```

Comment:
Hey @aryaroudi, please update this screenshot, and remember to update alt text for ALL images.
```suggestion
![Organization Settings - Details](../../images/cloud-application-manager/admin-assets1.png)
```